src/plugins/data-card/data-card-tile.tsx

```diff
@@ -28,6 +28,8 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer(function Dat
 
   const content = model.content as DataCardContentModelType;
   const dataSet = content.dataSet;
+  const linkedTiles = content.tileEnv?.sharedModelManager?.getSharedModelTiles(content.sharedModel);
+  const isLinked = linkedTiles && linkedTiles.length > 1;
```

Comment:
Could consider putting `isLinked` into a context rather than drilling it down through multiple layers of components.